openreview/conference/matching.py

```diff
@@ -110,6 +110,16 @@ def _create_edge_invitation(self, edge_id):
             'values-copied': edge_readers + ['{tail}']
         }
 
+        edge_head_type = 'Note'
+        edge_head_query = {
+            'invitation' : self.conference.get_blind_submission_id()
+        }
+        if 'Custom_Load' in edge_id:
+            edge_head_type = 'Group'
+            edge_head_query = {
+                'group' : edge_id.split('/-/')[0]
```

Comment:
The query object should be `{'id': <reviewer's group id>}`

Reply:
Thanks @zbialecki . Updated.